test/e2e/config/azwi.yaml

```diff
@@ -0,0 +1,272 @@
+apiVersion: v1
```

Comment:
why is this file needed?

Reply:
This config is used to deploy azwi webhook in e2e 
Updated the PR to include a comment also.